stacker/blueprints/asg.py

```diff
@@ -165,9 +165,18 @@ def create_load_balancer(self):
                     GetAtt(elb_name, 'DNSName')],
                 Condition="SetupELBDNS"))
 
+    def get_launch_configuration_parameters(self):
```

Comment:
I almost feel like this should return all of the parameters - and if people want to override any of them, they in theory could?